src/appsettings/settings.py

```diff
@@ -461,15 +463,17 @@ def default_value(self):
     @property
     def raw_value(self):
         """
-        Property to return the variable defined in ``django.conf.settings``.
+        Property to return the variable defined in ``django.conf.settings`` or in ``os.environ``.
 
         Returns:
-            object: the variable defined in ``django.conf.settings``.
+            object: the variable defined in ``django.conf.settings`` or in ``os.environ``.
```

Comment:
Environment should be noted first as used in code.

Reply:
Done.